### ISSUE TITLE
Include yaml in common language set

### ISSUE THREAD

**Issue by mojavelinux** [highlightjs/highlight.js#1632]
yaml is arguably one of the most ubiquitous and popular configuration languages. Requests for it come up over and over in the Asciidoctor project, where people often use the base highlight.js bundle. Therefore, I think it's reasonable to include yaml in the base set of languages bundled in the CDN.

I have no problem submitting a PR this change. But I want to make sure the idea is met with approval first.

**Comment by mojavelinux** [comment]
It's not clear to me where the languages included in the minimized file on the CDN are defined. Can someone enlighten me?

**Comment by ykfq** [comment]
`YAML` is widely used nowadays, especially in Ansible and Kubernetes manifests. Its would be great to include it into the `common` css file `default.min.css`

@mojavelinux 
Is it just put `Category: common` in the head of `src/languages/yaml.js` file?
Refer to this: [Remove awk.js from :common](https://github.com/highlightjs/highlight.js/commit/70308d7b15c78c36c1c692366d1f6dfb8a1dd628)

> "Common" is a special category, we use it to control the download size
of the default browser build.

**Comment by mojavelinux** [comment]
That is precisely my last question.

**Comment by mojavelinux** [comment]
The answer to that question is yes. I'll submit a PR.

**Comment by marcoscaceres** [comment]
> Requests for it come up over and over in the Asciidoctor project, where people often use the base highlight.js bundle. 

@mojavelinux sorry to make you do extra work, but can you provide links to the issues where people have asked for it. Want to get a sense of frequency (i.e., 3-5 requests to Asciidoctor per year or more).

**Comment by mojavelinux** [comment]
If this were a fringe language, I could understand. But frankly, this request is absurd. YAML is one of the most prominent configuration languages in the tech industry. Debating that fact is a waste of time.

Here's a list of high profile tools that use YAML and thus have configurations which frequently appear in documentation.

* Ansible
* Kubernetes
* Docker Compose
* Travis CI
* Circle CI
* GitLab CI
* AppVeyor
* Jekyll
* Spring Boot
* Monogo DB
* Cassandra
* RuboCop
* Zuul
* Coveralls
* Istanbul
* ESLint
* Prometheus
* DBUnit

Just a quick locate on my computer returns 32,000 entries for YAML files.

**Comment by ykfq** [comment]
> @mojavelinux sorry to make you do extra work, but can you provide links to the issues where people have asked for it. Want to get a sense of frequency (i.e., 3-5 requests to Asciidoctor per year or more).

I was intended to create an issue but I found there aready have one. Many maintainers and developers on github discourage user to create simillar issues, then I reused this one. So the requests per year maybe not accurate.

Another reason is that `YAML` is more likely used by operators and they do not care if YAML is in the common set of highlight.js of Web developing area.

YAML is populor and it would be great to include it into the common set.
